settings.php

```diff
@@ -975,38 +976,68 @@ function convertseconds($argument)
                                                     when enabling DNSSEC. A DNSSEC resolver test can be found
                                                     <a href="https://dnssec.vs.uni-due.de/" rel="noopener" target="_blank">here</a>.</p>
                                                 </div>
-                                                <strong>Conditional Forwarding</strong>
-                                                <p>If not configured as your DHCP server, Pi-hole won't be able to
+                                                <p>Validate DNS replies and cache DNSSEC data. When forwarding DNS
+                                                   queries, Pi-hole requests the DNSSEC records needed to validate
+                                                   the replies. If a domain fails validation or the upstream does not
+                                                   support DNSSEC, this setting can cause issues resolving domains.
+                                                   Use Google, Cloudflare, DNS.WATCH, Quad9, or another DNS
+                                                   server which supports DNSSEC when activating DNSSEC. Note that
+                                                   the size of your log might increase significantly
+                                                   when enabling DNSSEC. A DNSSEC resolver test can be found
+                                                   <a href="https://dnssec.vs.uni-due.de/" rel="noopener" target="_blank">here</a>.</p>
+                                                <h4>Conditional forwarding</h4>
```

Comment:
I think a line break here might be nicer, to break the content up a little.

Before:
![image](https://user-images.githubusercontent.com/1998970/85429670-fcc8ae00-b576-11ea-9647-b753b4283054.png)

```suggestion
                                                <br>
                                                <h4>Conditional forwarding</h4>
```

After:
![image](https://user-images.githubusercontent.com/1998970/85429703-08b47000-b577-11ea-96ae-af04e7d0636f.png)